101-multiple-windows-vms-with-common-script/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "_artifactsLocation": {
```

Comment:
remove the parameters that have defaultValues and use GEN values for the rest (see contribution guide)

Reply:
done